tests/unit/src/test/scala/tests/BuildTargetsLspSuite.scala

```diff
@@ -35,4 +38,22 @@ class BuildTargetsLspSuite
       )
     } yield ()
   }
+  test("metals/buildTargets") {
```

Comment:
Since `metals/buildTargets` is mainly just copying a copy of the internal state of build targets that Metals has, I wasn't sure exactly what would actually be useful to test. Mainly I have this here just as a sanity check to ensure that we are indeed getting back what we'd sort of expect without having to include all of the data that is actually coming when a user would call this.